hail/src/main/scala/is/hail/sparkextras/ContextRDD.scala

```diff
@@ -265,6 +265,32 @@ class ContextRDD[C <: AutoCloseable, T: ClassTag](
       preservesPartitioning),
     mkc)
 
+  def cmapPartitionsWithContext[U: ClassTag](f: (C, (C) => Iterator[T]) => Iterator[U]): ContextRDD[C, U] = {
```

Comment:
how do these changes fit in?

Reply:
They don't. Sorry, I broke this off of a bigger change trying to straighten out the region management in RVD, and these accidentally slipped in. Deleting.